operators/operator_sdk/osdk-upgrading-projects.adoc

```diff
@@ -0,0 +1,25 @@
+[id="osdk-upgrading-projects"]
+= Upgrading projects for newer Operator SDK versions
+include::modules/common-attributes.adoc[]
+:context: osdk-upgrading-projects
+
+:osdk_ver: v1.8.0
+:osdk_ver_n1: v1.3.0
+:product-version-n1: 4.7
+
+toc::[]
+
+{product-title} {product-title} supports Operator SDK {osdk_ver}. If you already have the {osdk_ver_n1} CLI installed on your workstation, you can upgrade the CLI to {osdk_ver} by xref:../../operators/operator_sdk/osdk-installing-cli.adoc#osdk-installing-cli[installing the latest version].
```

Comment:
Remove one of the two {product-title} thingies.

Reply:
2nd one was supposed to be `{product-version}`.